### PR TITLE
Dataframes support in Luna Studio

### PR DESCRIPTION
**Pull request by sylwiabr**
### Pull Request Description

This PR is adding the support for Dataframes in Luna Studio

### Important Notes

- Support the viisualizations for Dataframes
- Bump luna and StdLib with Lquery
- Fix the compilation on Windows

### Checklist
Please include the following checklist in your PR:

- [ x ] The documentation has been updated if necessary.
- [ x ] The code conforms to the [Luna Style Guide](https://github.com/luna/wiki/blob/master/code-style/01.general.md) if it is Haskell.
- [ x ] The code has been tested where possible.

### CODE REVIEW

File: env/Std/src/Base.luna [hunk 1]
```diff
@@ -1047,7 +1053,15 @@ class JSON:
     def toJSON: self
 
     # Dumps a `JSON` structure into a `Text` object.
-    def toText: renderJSON self
+    def toText: case self of
+        JSONNumber d: d.toText
+        JSONString s: s.escapeJSON
+        JSONArray  l: "[" + (l . map (x : x.toText) . intersperse "," . fold "" (+)) + "]"
```
Comment (ljkania) [inline env/Std/src/Base.luna]: Why don't we use `concat` instead of `fold "" (+)`?

Comment (ljkania) [reply env/Std/src/Base.luna]: Also, AFAIR: `map .toText` should work here too

File: env/Std/src/Base.luna [hunk 2]
```diff
@@ -1047,7 +1053,15 @@ class JSON:
...
+        JSONBool   b: case b of
+            True:  "true"
+            False: "false"
+        JSONNull: "null"
+        JSONObject m: "{" + (m . toList . map ((Tuple2 k v): k.escapeJSON + ": " + v.toText) . intersperse "," . fold "" (+)) + "}"
```
Comment (ljkania) [inline env/Std/src/Base.luna]: ditto

File: luna-studio/atom/lib/visualizers/data/base/config.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module.exports = function (type) {
-    var listVis  = (type.constructor === "List" || type.constructor === "Dataframe") ? [{name: "table", path: "table/table.html"}] : [];
+    var listVis  = (type.constructor === "List" || type.constructor === "Table" || type.constructor === "Column") ? [{name: "table", path: "table/table.html"}] : [];
```
Comment (ljkania) [inline luna-studio/atom/lib/visualizers/data/base/config.js]: Consider making a list of accepted constructors and using `contains` (or equivalent - I don't remember exact function name). There is a lot of repetition.

File: luna-studio/atom/lib/visualizers/data/base/table/table.js [hunk 4]
```diff
@@ -80,7 +80,18 @@
...
     };
     result += "</tr>";
-    data.forEach(function (row, ix) {
+    table = []
+
+    data.forEach(function(d, i) {
+      d.forEach(function(elem, idx) {
+        if (!table[idx]) {
+          table[idx] = []
+        }
```
Comment (ljkania) [inline luna-studio/atom/lib/visualizers/data/base/table/table.js]: `table[idx] = table[idx] || []`

File: env/Std/src/Base.luna (hunk 1 above)
Comment (sylwiabr) [reply env/Std/src/Base.luna]: We should consider this when we will be making changes in stdlib for time series in luna

File: env/Std/src/Base.luna (hunk 2 above)
Comment (sylwiabr) [reply env/Std/src/Base.luna]: like above

File: luna-studio/atom/lib/visualizers/data/base/config.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 module.exports = function (type) {
-    var listVis  = (type.constructor === "List" || type.constructor === "Table" || type.constructor === "Column") ? [{name: "table", path: "table/table.html"}] : [];
+    var listVis  = (["List", "Table", "Column"].includes(type.constructor)) ? [{name: "table", path: "table/table.html"}] : [];
```
Comment (ljkania) [inline luna-studio/atom/lib/visualizers/data/base/config.js]: Love it <3

**Pull request merged**